test/images/agnhost/Dockerfile

```diff
@@ -12,7 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM scratch
+FROM BASEIMAGE
+
+CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
+
+# from hostexec image
+# install necessary packages:
+# - curl, nc: used by a lot of e2e tests
+# - iproute2: includes ss used in NodePort tests
+RUN apk --update add curl netcat-openbsd iproute2 && rm -rf /var/cache/apk/*
+
+# from logs-generator
+ENV LOGS_GENERATOR_LINES_TOTAL 1
+ENV LOGS_GENERATOR_DURATION 1s
```

Comment:
non-blocking nit: I can't tell that these are actually consumed by `log_generator.go` so it's unclear why they're necessary

Reply:
True, it seems that the env vars are not consumed, the logs-generator binary was expecting those params as flags from the initial commit, yet the documentation was also talking about env vars. The tests also sets those env vars.